core/src/main/java/org/apache/calcite/plan/volcano/IterativeRuleDriver.java

```diff
@@ -22,11 +22,11 @@
 import org.slf4j.Logger;
 
 /***
- * <p>The algorithm executes repeatedly in a series of phases. In each phase
- * the exact rules that may be fired varies. The mapping of phases to rule
- * sets is maintained in the {@link #ruleQueue}.
+ * <p>The algorithm executes repeatedly.
+ * The exact rules that may be fired varies.
+ * The rule sets is maintained in the {@link #ruleQueue}.
```

Comment:
Remove this sentence. It is not needed anymore.

Reply:
Revised

Reply:
> Remove this sentence. It is not needed anymore.

Thanks for your suggestion, Revised.